### PR TITLE
strip compiling issues

### PR DESCRIPTION
**Pull request by YuriMzz**
Due to inconsistencies in some files, i was unable to compile with STRIP=on; i think i fixed them

### CODE REVIEW

File: src/fpx/relic_fp24_sqr.c [hunk 1]
```diff
@@ -37,7 +37,7 @@
 /* Public definitions                                                         */
 /*============================================================================*/
 
-#if FPX_RDC == BASIC || !defined(STRIP)
```
[the remark this answers is not in the document]
Comment (YuriMzz) [reply src/fpx/relic_fp24_sqr.c:40]: void fp24_sqr_cyc_lazyr(fp24_t c, const fp24_t a) {
	/* TODO: implement lazy reduction. */
	fp24_sqr_cyc_basic(c, a);
}

void fp24_sqr_pck_lazyr(fp24_t c, const fp24_t a) {
	/* TODO: implement lazy reduction. */
	fp24_sqr_pck_basic(c, a);
}
the lazy reduction implementation calls the basic function

File: include/relic_pc.h [hunk 2]
```diff
@@ -662,7 +662,7 @@ typedef RLC_CAT(RLC_GT_LOWER, t) gt_t;
  * @param[in] P					- the element to multiply.
  * @param[in] K					- the secret scalar.
  */
-#define g1_mul_key(R, P, K)		RLC_CAT(RLC_G1_LOWER, mul_lwreg)(R, P, K)
```
[the remark this answers is not in the document]
Comment (dfaranha) [reply include/relic_pc.h:665]: The right solution then is to make `ep_mul_lwreg` always compiled. `EP_MUL=BASIC` would arguably even be a configuration error.

File: src/fpx/relic_fp24_sqr.c (hunk 1 above)
[the remark this answers is not in the document]
Comment (dfaranha) [reply src/fpx/relic_fp24_sqr.c:40]: True, I forgot about these TODOs.

**Comment by dfaranha** [comment]
Oh, why did you close this PR?

**Comment by YuriMzz** [comment]
sorry, i'm not good at managing github stuff.

**Pull request closed without merging**